_layouts/song_collection_item.html

```diff
@@ -16,12 +15,16 @@ <h3 alt="{{ page.label | escape }}" class='item-label'>{{ page.label }}</h3>
       {% include osd_iiif_image_viewer.html manifest=page.manifest %}
     {% elsif page.full.size > 0 %}
       {% include simple_image_viewer.html full_image=page.full %}
+    {% else %}
+      {% include simple_image_viewer.html full_image='/assets/img/WOBLogo.png' %}
```

Comment:
Any ideas why this asset isn't rendering in the image viewer? I haven't had a chance to dig fully into it yet!